### DISCUSSION TITLE
Is this signature help and how do I get it?

### DISCUSSION THREAD

**Discussion by jasmine-nguyen** [saghen/blink.cmp#205] [Q&A] [answered]
Hi @Saghen , I'm trying to achieve [something like this](https://imgur.com/a/J5sQK6b), basically I want to see which package the suggestion is from so I can pick the right one. Is this signature help ? Because I have opted in for this in my [config here](https://github.com/jasmine-nguyen/neovim-config) but I could not get it to work. 

Could you let me know what I'm missing? Thank you.

**Comment by saghen** [comment, marked as answer]
You're looking for the `documentation` window which you can manually open with `Ctrl + Space` or have it automatically show with `windows.documentation.auto_show = true`. You also may want to watch #127 since `labelDetails` often show imports as well

**Reply by jasmine-nguyen** [reply]
That's exactly what I was looking for. Thank you !
